hubblestack/extmods/hubble_mods/win_reg.py

```diff
@@ -0,0 +1,223 @@
+# -*- encoding: utf-8 -*-
+"""
+Module for fetching registry values from windows registry
+
+Audit Example 1:
+---------------
+check_unique_id:
+  description: 'win_reg check'
+  tag: 'ADOBE-01'
+  implementations:
+    - filter:
+        grains: 'G@osfinger:Microsoft Windows Server 2016*'
+      hubble_version: '>3 AND <7 AND <8'
+      module: win_reg
+      items:
+        - args:
+            name: 'HKEY_LOCAL_MACHINE\Software\Policies\Microsoft\Windows\Explorer\NoDataExecutionPrevention'
+          comparator:
+            type: "dict"
+            match:
+              HKEY_LOCAL_MACHINE\Software\Policies\Microsoft\Windows\Explorer\NoDataExecutionPrevention:
```

Comment:
these backslashes are incorrect and will fail in python 3.7 or thereabouts.

Please add `r` to the `"""` (ie, `r"""`) or add another backslash to escape your escape. 

Python 3.6 and below can deal with this and does what you'd expect, but future pythons will not.

Reply:
In fact, this file won't load enough for me to even show the result under default warnings:

```
PYTHONPATH=. python -W default hubblestack/extmods/hubble_mods/win_reg.py 
  File "hubblestack/extmods/hubble_mods/win_reg.py", line 67
    """
SyntaxError: (unicode error) 'unicodeescape' codec can't decode bytes in position 426-427: malformed \N character escape
```